src/folly/FollyKeepaliveTimer.cpp

```diff
@@ -34,8 +34,7 @@ void FollyKeepaliveTimer::schedule() {
 void FollyKeepaliveTimer::sendKeepalive() {
   if (pending_) {
     stop();
-    connection_->closeWithError(
-        Frame_ERROR::connectionError("no response to keepalive"));
+    connection_->disconnect();
```

Comment:
I would like to keep the error message for the close. It would help with debugging.